### PR TITLE
[rp2040] Generate ADC driver info

### PR DESCRIPTION
**Pull request by cocasema**

### CODE REVIEW

File: tools/generator/raw-data-extractor/patches/rp2040.patch [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                   <name>usb_muxing_vbus_en</name>
+                   <value>9</value>
+                 </enumeratedValue>
++                <enumeratedValue>
++                  <name>adc_in3</name>
++                  <value>31</value>
++                </enumeratedValue>
+                 <enumeratedValue>
+                   <name>null</name>
+                   <value>31</value>
```
Comment (salkinium) [inline tools/generator/raw-data-extractor/patches/rp2040.patch:48]: Ok, the patch is smaller than I thought ;-P

**Review by salkinium — APPROVED** [review]
Thanks!

**Comment by salkinium** [comment]
(I'll wait until you've tested these changes in modm before merging)

**Comment by salkinium** [comment]
I've removed the flash memory section altogether since the device files are meant to describe the hardware as a read-only database and patching it is then breaking that idea.
If you're happy with this, I'll squash the commits.

**Comment by cocasema** [comment]
I'm totally happy with this :)

**Pull request merged**